# Pololu based motors

base_rot:
  pololu_id: 12
  motor_id: 0
  init: 1500
  min: 500
  max: 2500
  reverse: true

bottom_joint_1:
  pololu_id: 12
  motor_id: 6
  init: 1500
  min: 500
  max: 2500
  reverse: true

bottom_joint_2:
  pololu_id: 12
  motor_id: 2
  init: 1500
  min: 500
  max: 2500
  reverse: true

middle_joint:
  pololu_id: 12
  motor_id: 1
  init: 1500
  min: 500
  max: 2500
  reverse: true

wrist_joint:
  pololu_id: 12
  motor_id: 8
  init: 1500
  min: 500
  max: 2500
  reverse: true

wrist_rot:
  pololu_id: 12
  motor_id: 11
  init: 1500
  min: 1000
  max: 2000
  reverse: true

gripper:
  pololu_id: 12
  motor_id: 5
  init: 1500
  min: 1000
  max: 2000
  reverse: true
